src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutes.scala

```diff
@@ -38,26 +38,27 @@ abstract class LeoRoutes(val leonardoService: LeonardoService, val proxyService:
 
   def leoRoutes: Route =
     requireUserInfo { userInfo =>
-      path("isWhitelisted") {
-        get {
-          complete {
-            leonardoService.isWhitelisted(userInfo).map { _ =>
-              StatusCodes.OK
+      setTokenCookie(userInfo) {
+        path("isWhitelisted") {
```

Comment:
Note we should remove `isWhitelisted` (and related configs) eventually -- it's unused and undocumented in swagger.